src/commands/config/index.ts

```diff
@@ -0,0 +1,95 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { ChannelType } from 'discord-api-types';
+import { Category } from 'types/applicationCommand';
+export const data = new SlashCommandBuilder()
+    .setName('config')
+    .setDescription('Set your guilds config')
+    .addSubcommand(command =>
+        command
+            .setName('prefix')
+            .setDescription('Set the prefix for your guild')
+            .addStringOption(option =>
+                option
+                    .setName('prefix')
+                    .setDescription('The prefix you want to set')
+                    .setRequired(true)
+            )
+    )
+    .addSubcommand(command =>
+        command
+            .setName('antibot')
+            .setDescription('Set antibot factor! (Use 0 for disabled)')
+            .addNumberOption(option =>
+                option
+                    .setName('factor')
+                    .setDescription('Antibot factor')
+                    .setMinValue(0)
+                    .setMaxValue(100)
+                    .setRequired(true)
+            )
+    )
+    .addSubcommand(command =>
+        command
+            .setName('antibot-action')
+            .setDescription('Set antibot action')
+            .addStringOption(option =>
+                option
+                    .setName('action')
+                    .setDescription('Action to do when antibot is triggered')
+                    .addChoices([
+                        ['Ban', 'Ban'],
+                        ['Kick', 'Kick'],
+                        ['Timeout', 'timeout']
+                    ])
+                    .setRequired(true)
+            )
+    )
```

Comment:
what if we used subcommand groups for this?

Reply:
Do we have a subcommand grou handler